smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/ClientRuntimeTypes.kt

```diff
@@ -18,34 +18,14 @@ object ClientRuntimeTypes {
         val HttpClient = runtimeSymbol("HTTPClient")
         val HttpClientConfiguration = runtimeSymbol("HttpClientConfiguration")
         val Headers = runtimeSymbol("Headers")
-        val HttpStatusCode = runtimeSymbol("HttpStatusCode")
```

Comment:
Many unused types deleted, and some were moved to separate files for other smithy-swift modules.